beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/performance/AttestationPerformance.java

```diff
@@ -82,9 +87,10 @@ public int hashCode() {
   public String toString() {
     return String.format(
         "Attestation performance: "
-            + "expected %d, produced %d, included %d (%d%%), "
+            + "epoch %d, expected %d, produced %d, included %d (%d%%), "
```

Comment:
It would be worth switching all these `%d` to `%s`.  Using `%d` will encode the number using the current locale which is very confusing when the rest of the output is always English and has led to problems with charsets in the past.